std/math.d

```diff
@@ -4531,6 +4545,10 @@ public:
      /// A machine NaN was generated. (example: x = real.infinity * 0.0; )
      @property bool invalid() { return (flags & INVALID_MASK) != 0; }
 
+     /** A subnormal number was used in a calculation.
+     (example: x = real.min_normal / 2 + 1) */
+     @property bool subnormal() const { return (flags & SUBNORMAL_MASK) != 0; }
```

Comment:
> As far as I see, IeeeFlags only works on x86 and x86-64 at the moment. Everything else errors out with "not yet supported"

GDC has code for ARM as well: https://github.com/D-Programming-GDC/GDC/blob/master/libphobos/src/std/math.d#L4372 and LDC probably supports even more targets. The part of the code that has never been upstreamed is mostly GDC extended ASM so it's not useful for other compilers anyway.

We unfortunately don't have any guidelines yet how to deal with architecture specific features (there's a similar problem for `cpuid`). I guess we could add this function but we should at least make sure to explicitly document it as non-portable.